docs_src/src/pages/documentation/api_reference/openapi.mdx

```diff
@@ -215,6 +215,134 @@ app.include_router(subrouter)
 
 </CodeGroup>
 
+## Other Specification Params
+
+We support all the params mentioned in the latest OpenAPI specifications (https://swagger.io/specification/). See an example of using request body below.
+
+<CodeGroup title="Request Body">
+
+```python {{ title: 'untyped' }}
+@dataclass
+class Prop:
+    description: str
+    type: str
+
+
+@dataclass
+class PropObject:
+    prop_name: str
+    prop_body: Prop
+
+
+@dataclass
+class GetRequestSchema:
+    type: str
+    required: List[str]
+    properties: List[PropObject]
+
+    def __post_init__(self):
+        properties: dict = {}
+
+        for p in self.properties:
+            properties[p.prop_name] = p.prop_body
+
+        self.properties = properties
+
+
+get_request_schema_object = GetRequestSchema(
+    type="object",
+    required=["status"],
+    properties=[
+        PropObject(
+            prop_name="name",
+            prop_body=Prop(description="Updated name of the pet", type="String"),
+        ),
+        PropObject(
+            prop_name="status",
+            prop_body=Prop(description="Updated status of the pet", type="String"),
+        )
+    ]
+)
```

Comment:
@VishnuSanal , the developer is not supposed to write all this. just a class with key value pairs